src/core/src/op/softsign.cpp

```diff
@@ -90,6 +91,11 @@ bool ov::op::v9::SoftSign::evaluate(ov::TensorVector& outputs,
 
     const auto& in = inputs[0];
     auto& out = outputs[0];
-    out.set_shape(in.get_shape());
+
+    if (ov::util::is_dynamic_shape(out.get_shape())) {
+        out = Tensor(in.get_element_type(), in.get_shape());
+    } else {
+        out.set_shape(in.get_shape());
+    }
```

Comment:
why did you change it only here? I see that lots of evaluate methods set shape to the output tensor. On the other hand I think that evaluate should not work with DynamicTensors at all.

Oh I see. This is the ov::Tensor evaluate -- I see lots of HostTensor vectors.
Anyway -- it looks strange that we must check for dynamic shape before setting new shape.
@ilyachur please advice what should we do?

Reply:
New tensors don't support dynamic shapes. `{0, max_int}` is a WA for our internal code only in order to have a bridge between two APIs.
So I think you can just check the tensor size and call set_shape in case of empty tensor.

We don't need to use out internal API here. Maybe we need to modify `set_shape()` to support this WA

Reply:
Updated `set_shape` member function.